mne/gui/_coreg_gui.py

```diff
@@ -153,12 +167,57 @@ class CoregModel(HasPrivateTraits):
     point_distance = Property(depends_on=['transformed_mri_points',
                                           'transformed_hsp_points'])
 
+    # EEG point-related parameters
+    use_eeg_locations = Bool(True, label="Use EEG electrode locations "
+                             "as head shape points")
+
     # fit property info strings
     fid_eval_str = Property(depends_on=['lpa_distance', 'nasion_distance',
                                         'rpa_distance'])
     points_eval_str = Property(depends_on='point_distance')
 
     @cached_property
+    def _get_points(self):
+        if self.points_filter is None:
+            return self.hsp.inst_points
+        else:
+            return self.hsp.inst_points[self.points_filter]
+
+    def _get_points_filter(self):
```

Comment:
This method (`._get_points_filter()`) is identical to `_get_points()`, and is never called because `points_filter` is not a Property.